auth/store.go

```diff
@@ -103,6 +107,26 @@ func (as *authStore) UserAdd(r *pb.AuthUserAddRequest) (*pb.AuthUserAddResponse,
 	return &pb.AuthUserAddResponse{}, nil
 }
 
+func (as *authStore) UserDelete(r *pb.AuthUserDeleteRequest) (*pb.AuthUserDeleteResponse, error) {
+	plog.Noticef("deleting a user: %s", r.Name)
```

Comment:
this is going to be confusing. say we logged we are deleting a user and then there is a not found error happened, the logging is strange... it seems like the deleting never finished. probably just remove the deleting log. the deleted log at line 124 should be good enough.